bin/cli

```diff
@@ -84,34 +94,73 @@ const getChosenKitDependency = () => {
 
 (async () => {
   if (command === 'create') {
+    // Install as a two-stage bootstrap process.
+    //
+    // In stage one (`create`) we create an empty project folder and install
+    // govuk-prototype-kit and govuk-frontend, then bootstrap stage two from
+    // the newly installed package.
+    //
+    // In stage two (`init`) we do the actual setup of the starter files.
+    //
+    // Doing it this way means we can be sure the version of the cli matches
+    // the version of the kit the user ends up with. Try to put as much logic
+    // as possible into stage two; stage one should ideally be able to install
+    // any future version of the kit.
+
     const installDirectory = getInstallLocation()
     const kitDependency = getChosenKitDependency()
 
-    const copyFile = (fileName) => fs.copy(path.join(kitRoot, fileName), path.join(installDirectory, fileName))
-
     await fs.ensureDir(installDirectory)
     if ((await fs.readdir(installDirectory)).length > 0) {
       console.error(`Directory ${installDirectory} is not empty, please specify an empty location.`)
-      process.exit(3)
+      process.exitCode = 3
+      return
     }
 
-    await Promise.all([
-      fs.copy(path.join(kitRoot, 'prototype-starter'), installDirectory),
-      fs.writeFile(path.join(installDirectory, '.gitignore'), gitignore, 'utf8'),
-      fs.writeFile(path.join(installDirectory, '.npmrc'), npmrc, 'utf8'),
-      copyFile('LICENCE.txt')
-    ])
+    await fs.writeJson(path.join(installDirectory, 'package.json'), {}, packageJsonFormat)
 
     console.log('Creating your prototype')
     const dots = setInterval(() => {
       process.stdout.write('.')
     }, 500)
+
     await exec(`npm install ${kitDependency} govuk-frontend`, {
       stdio: 'inherit',
       cwd: installDirectory
     }, console.log)
+
     clearInterval(dots)
 
+    await exec(`npx govuk-prototype-kit init -- ${installDirectory}`, {
+      stdio: 'inherit',
+      cwd: installDirectory
+    }, console.log)
+  } else if (command === 'init') {
+    // `init` is stage two of the install process (see above), it should be
+    // called by `create` with the correct arguments.
+
+    if (additionalOptions[0] !== '--') {
```

Comment:
What's the reason for the double-hypen?  Is it just to make sure people don't use it by mistake?

Reply:
Yeah, exactly that 😁

Reply:
I'm wondering whether we should be more specific with it - something like `--internal-use-only` ... selfishly for the parser I've just created I'd love it if it was a key and value pair e.g. `---internal-use=true`.

If you think it's best as just `--` we can support that separately to the parser.

Reply:
Cool, I'm happy with that.